build.sbt

```diff
@@ -261,6 +264,29 @@ lazy val benchmarks = project
   .settings(commonSettings)
   .dependsOn(`play-jsonJVM`)
 
+lazy val docs = project
```

Comment:
A sub-project instead of a separated project with its `build.sbt`, `project/`, etc.